packages/account/src/Components/forms/idv-form.tsx

```diff
@@ -189,76 +189,81 @@ const IDVForm = ({
-                                                        placeholder={generatePlaceholderText(selected_doc)}
-                                                        value={values.document_number}
-                                                        onPaste={preventEmptyClipboardPaste}
-                                                        onBlur={handleBlur}
-                                                        onChange={handleChange}
-                                                        onKeyUp={(e: { target: HTMLInputElement }) =>
-                                                            onKeyUp(e, 'document_number')
-                                                        }
-                                                        required
-                                                        label={generatePlaceholderText(selected_doc)}
-                                                    />
-                                                    {values.document_type.additional?.display_name && (
+                                    {!(values.document_type.id === IDV_NOT_APPLICABLE_OPTION.id) && (
+                                        <fieldset
+                                            className={classNames({
+                                                'proof-of-identity__fieldset-input': !hide_hint,
+                                            })}
+                                        >
+                                            <Field name='document_number'>
+                                                {({ field }: FieldProps) => (
+                                                    <React.Fragment>
                                                         <Input
                                                             {...field}
-                                                            name='document_additional'
+                                                            name='document_number'
                                                             bottom_label={
-                                                                values.document_type.additional &&
+                                                                values.document_type &&
                                                                 getExampleFormat(
-                                                                    values.document_type.additional?.example_format
+                                                                    values.document_type.example_format ?? ''
                                                                 )
                                                             }
                                                             disabled={
                                                                 !values.document_type.id ||
                                                                 values.document_type.id === IDV_NOT_APPLICABLE_OPTION.id
```

Comment:
This code block wont be rendered if values.document_type.id === IDV_NOT_APPLICABLE_OPTION.id right?
do we need this disable condition

Reply:
@amina-deriv , no need of this, thank you as well 👍